docs/dataformats/file_formats.rst

```diff
@@ -194,6 +194,9 @@ XML
 
 **XML** files (see `Wikipedia <http://en.wikipedia.org/wiki/Xml>`__)
 
+GammaLib / ctools uses an "observation definition" XML format described
+`here <http://gammalib.sourceforge.net/user_manual/modules/obs.html#describing-observations-using-xml>`__.
```

Comment:
This comment hasn't been addressed, please point to http://cta.irap.omp.eu/gammalib-devel/  instead.

(What I usually do before asking for final review and merge on a PR is to do a final review on Github myself, i.e. look at the `Conversation`, `Commits` and `Files changed` tabs. Github only shows inline comments where the corresponding line hasn't been updated by a later commit. Sometimes this means that one comment hasn't been addressed, sometimes the conclusion of the discussion was that no change is needed.